chains/incentiv/metadata.yaml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: http://explorer.incentive.io/api
+    family: other
+    name: Incentiv Explorer
+    url: http://explorer.incentive.io
```

Comment:
These links don't work for me i get a go daddy page

Reply:
ok having a better look, shouldn't these be `https://explorer.incentiv.io/` ?i think you have an extra `e` at the end